package.json

```diff
@@ -74,6 +74,8 @@
     "*": "prettier -w -u"
   },
   "dependencies": {
+    "@floating-ui/core": "^0.7.3",
+    "@floating-ui/dom": "^0.5.3",
```

Comment:
this is a dependencies or a dev dependencies?

Reply:
Dependency. It's not types, there's functionnality.